awx/settings/production.py

```diff
@@ -46,16 +42,6 @@
 
 AWX_ISOLATED_USERNAME = 'awx'
 
-LOGGING['handlers']['tower_warnings']['filename'] = '/var/log/tower/tower.log'  # noqa
```

Comment:
`LOG_ROOT` is now set to `/var/log/tower` instead of dynamically being set to the path where the awx code lives. Therefore, these overrides are not needed. Instead, the `LOGGING['handlers']` settings already defined in defaults.py will be the authority of record. A single place.